tests/policies/policy_helpers/test_helpers.py

```diff
@@ -0,0 +1,12 @@
+import tensorflow as tf
+from policies.policy_helpers.helpers import ActionSampler
+from policies.gac.gac_networks import AutoRegressiveStochasticActor
+
+def test_ActionSampler():
```

Comment:
Not exactly standard, but, if it works, it works.

Reply:
Yeah. Definitely not standard. Just for me to see it runs and the tensor shapes match.